crates/api/src/local_user.rs

```diff
@@ -103,8 +105,9 @@ impl Perform for Login {
     }
 
     // Return the jwt
+    let jwt_secret = blocking(context.pool(), move |conn| Secrets::read_jwt_secret(conn)).await??;
     Ok(LoginResponse {
-      jwt: Claims::jwt(local_user_view.local_user.id.0, context.pool()).await?,
+      jwt: Claims::jwt(local_user_view.local_user.id.0, jwt_secret.as_ref())?,
```

Comment:
You shouldn't have to pass in a connection here. Check out how Settings does this with an init: https://github.com/LemmyNet/lemmy/blob/main/crates/utils/src/settings/mod.rs

IE, on server startup (or tests), you read the secret, put it in lazy_static, and never do another DB call for it. If you're calling blocking for anything past the initial init, its not setup correctly.